python/GafferTest/NodeAlgoTest.py

```diff
@@ -75,6 +75,25 @@ def testUserDefaults( self ) :
 		self.assertEqual( node3["op1"].getValue(), 0 )
 		self.assertFalse( Gaffer.NodeAlgo.isSetToUserDefault( node["op1"] ) )
 
+	def testUserDefaultConnectedToCompute( self ) :
+
+		srcNode = GafferTest.AddNode()
+		node = GafferTest.AddNode()
+
+		Gaffer.Metadata.registerValue( GafferTest.AddNode.staticTypeId(), "op1", "userDefault", IECore.IntData( 7 ) )
```

Comment:
The `staticTypeId()` part of this is unnecessary and unpythonic. Since it looks like you got it from other parts of this file, could you add another commit that fixes the other occurrences too please? I've been trying to eradicate them and I'm not sure how these crept in.

Reply:
Ahh neat, thanks. Updated the ones I could find [here](https://github.com/GafferHQ/gaffer/pull/3895/commits/037d76f645791df6c5f2c874144d702b5ce611d6).